src/handlers/motions/motionVoteRevealed/index.ts

```diff
@@ -0,0 +1,54 @@
+import { ContractEvent, VoterRecord } from '~types';
+import { getVotingClient, verbose } from '~utils';
+import {
+  getMotionDatabaseId,
+  getMotionFromDB,
+  updateMotionInDB,
+} from '../helpers';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const {
+    contractAddress: colonyAddress,
+    args: { motionId, voter, vote },
+  } = event;
+
+  const votingClient = await getVotingClient(colonyAddress);
+  const { chainId } = await votingClient.provider.getNetwork();
+  const motionDatabaseId = getMotionDatabaseId(
+    chainId,
+    votingClient.address,
+    motionId,
+  );
+  const revealedMotion = await getMotionFromDB(colonyAddress, motionDatabaseId);
+
+  if (revealedMotion) {
```

Comment:
As in the other one, should we log if not found?

Reply:
Gets logged inside getMotionFromDB